packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -178,6 +175,14 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
                 )}
               </Cell>
             )}
+            <Row className={removePaddingStyles}>
```

Comment:
took me a lot of time to figure out that nesting rows creates a Collapse button. How did you find it? it's not even documented.

but then it does not look like other accordians we have in Compass, where the whole title is clickable.

Reply:
yeah I've searched threads in the leafygreen channel

Reply:
> where the whole title is clickable

mmm that's a good point .. should be fine, the problem with the accordion is that the cell content is really tough to maintain aligned, and It was already messed up after the LG update